tests/k8s_testcases/example_policies/services1/services1-scheme.yaml

```diff
@@ -5,25 +5,25 @@ networkConfigList:
   - name: ip
     networkPolicyList:
       - policies.yaml
-    expectedWarnings: 0
+    expectedWarnings: 1
```

Comment:
why has the expected warning changed on this test?

Reply:
because services in this test include `status` field .
I've added this field to the allowed keys with code 3 , so the new warning is :
`status is not yet supported inside Service,ignoring this key`

I found 'status' field in k8sIngress and Service objects in our tests, so I've added it to the ` allowed_keys` in case of 'k8s' type, otherwise, we'll get an error of `bad policy keys` (as an expected key to find)
 **maybe I should add this key with code 0 instead of 3?** 

However, I've added `status` as an allowed field for all `k8s` objects  since I found in k8s reference that this field is now supported in addition to `kind, spec, metadata and apiVersion` [(k8s_ref_example)](https://kubernetes.io/docs/reference/kubernetes-api/policy-resources/network-policy-v1/)

Reply:
yes, it should be with code 0 instead of 3.
It was already supported in k8s netpols from a previous commit: 
https://github.com/IBM/network-config-analyzer/commit/9ff2b173561ab7958a7d13eb0db125b7e2e8a3f7